packages/slate-editor/src/extensions/file-attachment/components/FileAttachment.module.scss

```diff
@@ -2,30 +2,12 @@
 @import "styles/mixins";
 @import "styles/variables";
 
-.file-attachment-element {
-    @include editor-v4-block($void: false);
-
-    border: 1px solid $legacy-color-grey-250;
-    border-radius: $border-radius-small;
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.04);
-
-    &--active {
-        @include editor-v4-block-active;
-    }
-
-    &--unstyled {
-        box-shadow: none;
-    }
-}
-
-.content {
-    @include editor-v4-void-element();
```

Comment:
The problem was that the `.file-attachment-element` here was actually styled as a top-level frame. Which was unnecessary, as we've wrapped it with `EditorBlock` already.